test/causal-cluster-compose.yml

```diff
@@ -12,8 +12,11 @@ services:
       - lan
     environment:
       - NEO4J_ACCEPT_LICENSE_AGREEMENT=yes
-      - NEO4J_AUTH=none
+      - NEO4J_AUTH=neo4j/neo
       - NEO4J_dbms_mode=CORE
+      - NEO4J_causalClustering_discoveryAdvertisedAddress=core1:5000
+      - NEO4J_causalClustering_transactionAdvertisedAddress=core1:6000
+      - NEO4J_causalClustering_raftAdvertisedAddress=core1:7000
```

Comment:
IMO this test is explicitly checking that the default behaviour hasn't regressed. I don't think these extra AdvertisedAddress values should be added.

Reply:
@eastlondoner is correct. these settings were purposefully left out of the first core to make sure that default settings are good. They are included in subsequent cores to verify that they can be overwritten.

Reply:
ack